docassemble_base/docassemble/base/core.py

```diff
@@ -4162,12 +4162,14 @@ def get_pdf_fields(self):
         """Returns a list of fields that exist in the PDF document"""
         results = list()
         import docassemble.base.pdftk
-        for item in docassemble.base.pdftk.read_fields(self.path()):
-            the_type = re.sub(r'[^/A-Za-z]', '', str(item[4]))
-            if the_type == 'None':
-                the_type = None
-            result = (item[0], '' if item[1] == 'something' else item[1], item[2], item[3], the_type, item[5])
-            results.append(result)
+        all_items = docassemble.base.pdftk.read_fields(self.path())
+        if all_items is not None:
+            for item in all_items: 
+                the_type = re.sub(r'[^/A-Za-z]', '', str(item[4]))
+                if the_type == 'None':
+                    the_type = None
+                result = (item[0], '' if item[1] == 'something' else item[1], item[2], item[3], the_type, item[5])
+                results.append(result)
         return results
```

Comment:
Shouldn't this be indented one level?

Reply:
Do you mean this return statement on 4173? (Github is bad at pointing to the exact line)  And I don't think so, otherwise the function just wouldn't return at all.

Reply:
Sorry--I missed line 4163! I thought `results` was undefined if `all_items` was `None`